subprojects/docs/src/docs/userguide/nativeBinaries.adoc

```diff
@@ -0,0 +1,943 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[native_software]]
+== Building native software
+
+
+[NOTE]
+====
+
+Support for building native software is currently <<feature_lifecycle,incubating>>. Please be aware that the DSL, APIs and other configuration may change in later Gradle versions.
+
+====
+
+The native software plugins add support for building native software components, such as executables or shared libraries, from code written in C++, C and other languages. While many excellent build tools exist for this space of software development, Gradle offers developers its trademark power and flexibility together with dependency management practices more traditionally found in the JVM development space.
+
+The native software plugins make use of the Gradle <<software_model,software model>>.
+
+
+[[sec:native_features]]
+=== Features
+
+The native software plugins provide:
+
+* Support for building native libraries and applications on Windows, Linux, OS X and other platforms.
+* Support for several source languages.
+* Support for building different variants of the same software, for different architectures, operating systems, or for any purpose.
+* Incremental parallel compilation, precompiled headers.
+* Dependency management between native software components.
+* Unit test execution.
+* Generate Visual studio solution and project files.
+* Deep integration with various tool chain, including discovery of installed tool chains.
+
+
+[[sec:supported_languages]]
+=== Supported languages
+
+The following source languages are currently supported:
+
+* C
+* C++
+* Objective-C
+* Objective-C++
+* Assembly
+* Windows resources
+
+
+[[native-binaries:tool-chain-support]]
+=== Tool chain support
+
+Gradle offers the ability to execute the same build using different tool chains. When you build a native binary, Gradle will attempt to locate a tool chain installed on your machine that can build the binary. You can fine tune exactly how this works, see <<native_binaries:tool_chain>> for details.
+
+The following tool chains are supported:
+
+[cols="a,a,a", options="header"]
```

Comment:
⭕️  Nitpick: this causes different table layout than what we had before. 

#### Old
![screen shot 2017-09-05 at 9 52 46 am](https://user-images.githubusercontent.com/51534/30073046-39a18e40-9221-11e7-9af3-b904cff607c0.png)

#### New 
![screen shot 2017-09-05 at 9 52 36 am](https://user-images.githubusercontent.com/51534/30073056-3f80beb2-9221-11e7-9d20-a8ed0033f55b.png)

Note that in general, Asciidoc seems to generate `<p>` tags within every table cell, adding unnecessary padding to the bottom. This should not be addressed here, but wanted to mention.

Reply:
Any ideas how to fix that?